docs/proposals/20200329-join.md

```diff
@@ -0,0 +1,48 @@
+# KubeOne Join
```

Comment:
I'd rename this proposal to something like `KubeOne Static Workers` or `Provisioning Worker Nodes Using KubeOne` to make it clear what this proposal is about.